tests/DependabotHelper.Tests/ApiTests.cs

```diff
@@ -958,6 +1200,7 @@ public async Task Can_Get_Pull_Requests_External_Reviewer_Cannot_Approve(string
         var actualPullRequest = actual.All[0];
 
         actualPullRequest.ShouldNotBeNull();
+        actualPullRequest.CanApprove.ShouldBeTrue();
```

Comment:
Just going off the name of the test `Can_Get_Pull_Requests_External_Reviewer_Cannot_Approve`, should CanApprove be False here?

Reply:
It's true in this context because the calling user is "you" (which is the user `john-smith`), not the other person who left that review.

The authorized API user is allowed to review the PR because it has no reviews against it that count and they haven't reviewed it themselves.